ocs_ci/ocs/defaults.py

```diff
@@ -22,6 +22,7 @@
 KUBECONFIG_LOCATION = 'auth/kubeconfig'  # relative from cluster_dir
 API_VERSION = "v1"
 CEPHFILESYSTEM_NAME = 'ocsci-cephfs'
+DEFAULT_STORAGECLASS_NAME = 'rook-ceph-block'
```

Comment:
we can avoid using this sc because it belongs to flexvolume

Reply:
thanks, will fix

Reply:
Added by mistake, will remove